docs/setup/ubuntu2204.md

```diff
@@ -46,7 +46,7 @@ otherwise just use the base (barebones) install
 
 1. We also recommend installing a few other packages:
 
-        sudo apt install -y python3-colcon-common-extensions python3-rosdep ros-humble-rmw-cyclonedds-cpp ros-humble-irobot-create-msgs
+        sudo apt install -y build-essential python3-colcon-common-extensions python3-rosdep ros-humble-rmw-cyclonedds-cpp #ros-humble-irobot-create-msgs
```

Comment:
I suggest to remove `ros-humble-irobot-create-msgs` rather than commenting it.
It may be confusing for people (although there's a warning below)

Reply:
leaving this since I'll get to un-comment it and remove the warning on Friday